app/templates/components/file-picker.hbs

```diff
@@ -11,9 +11,15 @@
 {{/if}}
 
 {{#if progress}}
-  <div class="file-picker__progress">
-    <span class="file-picker__progress__value" style={{progressStyle}}></span>
-  </div>
+  {{#if showProgress}}
+    {{#if isProgressSupported}}
+      <progress value={{progressValue}} max="100">{{progress}} %</progress>
```

Comment:
You are right about the migration but I would like to stick with BEM and not encourage people to use tag selectors.
I plan to drop old ember versions with the new release and add a migration guide.